core/src/main/scala/cats/Traverse.scala

```diff
@@ -133,7 +133,6 @@ import scala.annotation.implicitNotFound
 }
 
 object Traverse {
-  implicit def catsTraverseForEither[A]: Traverse[Either[A, *]] = cats.instances.either.catsStdInstancesForEither[A]
```

Comment:
Why is this line gone?

Reply:
I think this line is a mistake (it should be in `UnorderedFoldable`) and it actually caused an issue when compiling with dotty 0.25. I must've checked this in by mistake, I'll include it in a followup PR for dotty.

Reply:
Alright, but isn't this a binary-breaking change?